django/contrib/postgres/lookups.py

```diff
@@ -60,3 +60,24 @@ def process_lhs(self, qn, connection):
             self.lhs = SearchVector(self.lhs)
         lhs, lhs_params = super(SearchLookup, self).process_lhs(qn, connection)
         return lhs, lhs_params
+
+
+class TrigramSimilar(PostgresSimpleLookup):
+    lookup_name = 'trigram_similar'
+    operator = '%%'
+
+
+class TrigramDistance(PostgresSimpleLookup):
```

Comment:
I modelled it on the distance lookups in geodjango, which take multiple arguments: https://docs.djangoproject.com/en/1.9/ref/contrib/gis/geoquerysets/#distance-lookups – that's geographic rather than trigram distance, of course, but I thought it was consistent with them. Perhaps as a follow on to https://code.djangoproject.com/ticket/24214 they are going to be deprecated in future?

Just so I'm clear then, sorry, would `filter(foo__trigram_distance=("Test", 0.5))` be the same as `filter(foo=TrigramDistance("Test", 0.5))`? If so, then that's fine, I just thought they differed somehow.